### PR TITLE
Data structures review

### PR DESCRIPTION
**Pull request by maxspencer**
This PR relates to issue #11.

I've not quite finished the data structures one, but there's a good bit to be going on with, I'll finish the rest on the train tomorrow. This was much easier than the functions lesson!

### CODE REVIEW

**Comment by NataliaLKB** [comment]
👍

File: source/data_structures.rst [hunk 1]
```diff
@@ -1,108 +1,124 @@
...
 
-.. code-block:: python
+To create a list from some other objects use square brackets like so::
 
-    >>> ['John', 'Eric', 'Michael', 'Terry']          # literal
-    >>> list('abc')                                   # `list` transform a string
-    ['a', 'b', 'c'] 
-    >>> type([1, 2, 3])
-    <class 'list'>
+    >>> monty_python = ['John', 'Terry G', 'Eric', 'Michael', 'Terry J', 'Graham']
```
Comment (gtrufitt) [inline source/data_structures.rst:22]: Did we learn variables already? I like this better but just checking this is a later section (pretty sure it is). Maybe explicitly state we're assigning to a variable.

**Comment by gtrufitt** [comment]
🆒

**Comment by NataliaLKB** [comment]
👍 🚢 !

**Pull request merged**